tailwind.config.ts

```diff
@@ -12,6 +12,7 @@ export default {
       },
       boxShadow: {
         default: "8px 8px 0px 0px #4A4A51",
```

Comment:
nit: if this is `DEFAULT`, the style would become `shadow` instead of having to be `shadow-default`

Reply:
I think it got overwritten after some merge